hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,19 +4,17 @@
 
 This page explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-See the [bootmodes documentation](README.md) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
+See the [bootmodes documentation](README.md) for the boot sequence and alternative boot modes (network, USB device, GPIO or SD boot).
 
 Note that "USB mass storage boot" is different from "USB device boot mode". The [device boot mode](device.md) allows a Raspberry Pi connected to a computer to boot using files on that computer.
 
 For devices that are not supported, an alternative is to use the 'special bootcode.bin-only boot mode' as described [here](README.md). This still requires/boots from an SD-card, but `bootcode.bin` is the only file read from the SD-Card.
 
-## Raspberry Pi 2B v1.2, 3A+, 3B, CM3
+## Raspberry Pi 2B v1.2, 3A+, 3B, Compute Module 3
 
-On the Raspberry Pi 2B v1.2, 3A+, 3B, CM3, first USB [host boot mode](host.md) should be enabled. This is to allow "mass storage boot" and ["network boot"](net.md) (Network boot not supported on 3A+).
+On the Raspberry Pi 2B v1.2, 3A+, 3B, and Compute Module 3, you must first enable [USB host boot mode](host.md). This is to allow USB mass storage boot, and [network boot](net.md). Note that network boot not supported on Raspberry Pi 3A+.
```

Comment:
The added comma doesn't seem necessary?

" network boot not supported" -> " network boot is not supported"